From 6e82a8555980af9eaadce7d011d891d556f6307d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jonah=20Br=C3=BCchert?= <jbb.prv@gmx.de>
Date: Mon, 25 May 2020 18:43:21 +0200
Subject: [PATCH 01/21] StartupFeedback: calculate background color from icon

---
 components/qml/StartupFeedback.qml | 17 ++++++++++++-----
 1 file changed, 12 insertions(+), 5 deletions(-)

diff --git a/components/qml/StartupFeedback.qml b/components/qml/StartupFeedback.qml
index 998b4d8..0fc7465 100644
--- a/components/qml/StartupFeedback.qml
+++ b/components/qml/StartupFeedback.qml
@@ -21,31 +21,36 @@ import QtQuick 2.12
 import QtQuick.Layouts 1.1
 import QtQuick.Window 2.2
 import org.kde.plasma.core 2.0 as PlasmaCore
-import org.kde.plasma.components 2.0 as PlasmaComponents
-import org.kde.plasma.extras 2.0 as PlasmaExtras
 import QtGraphicalEffects 1.12
 
+import org.kde.kirigami 2.13 as Kirigami
+
 pragma Singleton
 
 Window {
     id: window
 
-    function open(splashIcon, title, color, x, y, sourceIconSize) {
+    Kirigami.ImageColors {
+        id: colorGenerator
+
+        source: icon.source
+    }
+
+    function open(splashIcon, title, x, y, sourceIconSize) {
         iconParent.scale = sourceIconSize/iconParent.width;
         background.scale = 0;
         background.x = -window.width/2 + x
         background.y = -window.height/2 + y
         window.title = title;
         icon.source = splashIcon;
-        background.color = color;
         background.state = "open";
     }
 
     property alias state: background.state
     property alias icon: icon.source
+
     width: Screen.width
     height: Screen.height
-    color: "transparent"
     onVisibleChanged: {
         if (!visible) {
             background.state = "closed";
@@ -89,6 +94,8 @@ Window {
         width: window.width
         height: window.height
 
+        color: colorGenerator.dominant
+
         state: "closed"
 
         states: [
-- 
2.26.2

