From c702803fc6eec9e8218df967a1157f84a3fdd4ce Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jonah=20Br=C3=BCchert?= <jbb.prv@gmx.de>
Date: Mon, 25 May 2020 18:52:03 +0200
Subject: [PATCH 02/21] StartupFeedback: Add optional color argument

---
 components/qml/StartupFeedback.qml | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/components/qml/StartupFeedback.qml b/components/qml/StartupFeedback.qml
index 0fc7465..14bc9ba 100644
--- a/components/qml/StartupFeedback.qml
+++ b/components/qml/StartupFeedback.qml
@@ -36,13 +36,18 @@ Window {
         source: icon.source
     }
 
-    function open(splashIcon, title, x, y, sourceIconSize) {
+    function open(splashIcon, title, x, y, sourceIconSize, color) {
         iconParent.scale = sourceIconSize/iconParent.width;
         background.scale = 0;
         background.x = -window.width/2 + x
         background.y = -window.height/2 + y
         window.title = title;
         icon.source = splashIcon;
+
+        if (color !== undefined) {
+            background.color = color
+        }
+
         background.state = "open";
     }
 
-- 
2.26.2

