From 4b954a4db2baee059398cd60409ee0f32116d0b9 Mon Sep 17 00:00:00 2001
From: Marco Martin <notmart@gmail.com>
Date: Thu, 16 Jul 2020 13:37:37 +0200
Subject: [PATCH 10/21] fix explorer layout

---
 shell/contents/explorer/WidgetExplorer.qml | 20 ++++++++++++++------
 1 file changed, 14 insertions(+), 6 deletions(-)

diff --git a/shell/contents/explorer/WidgetExplorer.qml b/shell/contents/explorer/WidgetExplorer.qml
index a592481..78f18a2 100644
--- a/shell/contents/explorer/WidgetExplorer.qml
+++ b/shell/contents/explorer/WidgetExplorer.qml
@@ -79,7 +79,7 @@ Controls.Drawer {
             z: 1
             anchors.bottom: parent.bottom
             x: root.horizontal ? parent.width - width : 0
-            width: units.iconSizes.huge
+            width: units.iconSizes.large
             height: width
 
             icon.name: "view-filter"
@@ -105,10 +105,18 @@ Controls.Drawer {
                 clip: false
                 Layout.fillWidth: root.horizontal
                 Layout.fillHeight: !root.horizontal
-                Layout.column: 1
-                Layout.row: root.horizontal ? 2 : 1
+                Layout.column: 0
+                Layout.row: root.horizontal ? 1 : 0
                 implicitHeight: units.gridUnit * 2
                 implicitWidth: units.gridUnit * 8
+                opacity: categoriesView.shouldBeVisible
+
+                Behavior on opacity {
+                    NumberAnimation {
+                        duration: units.longDuration
+                        easing.type: Easing.InOutQuad
+                    }
+                }
 
                 ListView {
                     clip: false
@@ -136,10 +144,10 @@ Controls.Drawer {
                 clip: false
                 Layout.fillWidth: root.horizontal
                 Layout.fillHeight: !root.horizontal
-                Layout.row: 1
-                Layout.column: root.horizontal ? 1 : 2
+                Layout.row: 0
+                Layout.column: root.horizontal ? 0 : 1
                 implicitWidth: delegateSize + units.gridUnit
-                implicitHeight: delegateSize + units.gridUnit*3
+                implicitHeight: delegateSize + units.gridUnit * 3
 
                 ListView {
                     id: appletsList
-- 
2.26.2

