From 1c0ed8551d136e31e067c9b10f3e006b85f4086e Mon Sep 17 00:00:00 2001
From: Marco Martin <notmart@gmail.com>
Date: Wed, 22 Jul 2020 14:52:46 +0200
Subject: [PATCH 12/21] restore previous hacks

---
 components/fullscreenoverlay.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/components/fullscreenoverlay.cpp b/components/fullscreenoverlay.cpp
index b7a5bd9..ab2ad22 100644
--- a/components/fullscreenoverlay.cpp
+++ b/components/fullscreenoverlay.cpp
@@ -99,8 +99,8 @@ bool FullScreenOverlay::event(QEvent *e)
         QPlatformSurfaceEvent *pe = static_cast<QPlatformSurfaceEvent*>(e);
 
         if (pe->surfaceEventType() == QPlatformSurfaceEvent::SurfaceCreated) {
-            //KWindowSystem::setState(winId(), NET::SkipTaskbar | NET::SkipPager | NET::FullScreen);
-           // setWindowStates(Qt::WindowFullScreen);
+            KWindowSystem::setState(winId(), NET::SkipTaskbar | NET::SkipPager | NET::FullScreen);
+            setWindowStates(Qt::WindowFullScreen);
             if (!m_acceptsFocus) {
                 setFlags(flags() | Qt::FramelessWindowHint|Qt::WindowDoesNotAcceptFocus);
                 //KWindowSystem::setType(winId(), NET::Dock);
-- 
2.26.2

