From 6db2dec1e1a38efd17b4b1bf67928e180ee1bb08 Mon Sep 17 00:00:00 2001
From: Marco Martin <notmart@gmail.com>
Date: Tue, 28 Jul 2020 11:41:21 +0200
Subject: [PATCH 15/21] prettier startup feedback

---
 components/qml/StartupFeedback.qml | 28 +++++++++++++++++++++++++---
 1 file changed, 25 insertions(+), 3 deletions(-)

diff --git a/components/qml/StartupFeedback.qml b/components/qml/StartupFeedback.qml
index b0555f9..0e3ca61 100644
--- a/components/qml/StartupFeedback.qml
+++ b/components/qml/StartupFeedback.qml
@@ -25,15 +25,16 @@ import QtGraphicalEffects 1.12
 
 import org.kde.kirigami 2.13 as Kirigami
 
+import org.kde.plasma.private.nanoshell 2.0 as NanoShell
+
 pragma Singleton
 
-Window {
+NanoShell.FullScreenOverlay {
     id: window
 
+    property alias backgroundColor: background.color
     Kirigami.ImageColors {
         id: colorGenerator
-
-        source: icon.source
     }
 
     function open(splashIcon, title, x, y, sourceIconSize, color) {
@@ -43,6 +44,7 @@ Window {
         background.y = -window.height/2 + y
         window.title = title;
         icon.source = splashIcon;
+        colorGenerator.source = splashIcon;
 
         if (color !== undefined) {
             // Break binding to use custom color
@@ -93,15 +95,24 @@ Window {
             source: icon
         }
     }
+
     Rectangle {
         id: background
         width: window.width
         height: window.height
+        radius: Kirigami.Units.gridUnit
 
         color: colorGenerator.dominant
 
         state: "closed"
 
+        Rectangle {
+            id: fill
+            z: -1
+            anchors.fill: parent
+            opacity: 0
+            color: Kirigami.Theme.backgroundColor
+        }
         states: [
             State {
                 name: "closed"
@@ -109,6 +120,10 @@ Window {
                     target: window
                     visible: false
                 }
+                PropertyChanges {
+                    target: fill
+                    opacity: 0
+                }
             },
             State {
                 name: "open"
@@ -159,6 +174,13 @@ Window {
                             easing.type: Easing.InOutQuad
                         }
                     }
+                    OpacityAnimator {
+                        target: fill
+                        from: 0
+                        to: 1
+                        duration: units.shortDuration
+                        easing.type: Easing.InOutQuad
+                    }
                 }
             }
         ]
-- 
2.26.2

