From 397abcd9ac3c2c83bd1129bf164268f2a96dc839 Mon Sep 17 00:00:00 2001
From: Marco Martin <notmart@gmail.com>
Date: Thu, 30 Jul 2020 12:39:49 +0200
Subject: [PATCH 16/21] not rounded anymore

---
 components/qml/StartupFeedback.qml | 10 +---------
 1 file changed, 1 insertion(+), 9 deletions(-)

diff --git a/components/qml/StartupFeedback.qml b/components/qml/StartupFeedback.qml
index 0e3ca61..32197d9 100644
--- a/components/qml/StartupFeedback.qml
+++ b/components/qml/StartupFeedback.qml
@@ -29,7 +29,7 @@ import org.kde.plasma.private.nanoshell 2.0 as NanoShell
 
 pragma Singleton
 
-NanoShell.FullScreenOverlay {
+Window {
     id: window
 
     property alias backgroundColor: background.color
@@ -100,19 +100,11 @@ NanoShell.FullScreenOverlay {
         id: background
         width: window.width
         height: window.height
-        radius: Kirigami.Units.gridUnit
 
         color: colorGenerator.dominant
 
         state: "closed"
 
-        Rectangle {
-            id: fill
-            z: -1
-            anchors.fill: parent
-            opacity: 0
-            color: Kirigami.Theme.backgroundColor
-        }
         states: [
             State {
                 name: "closed"
-- 
2.26.2

